### ISSUE TITLE
What is the future of Centaury?

### ISSUE THREAD

**Issue by ghost** [Feodor2/Centaury#47]
It seems MyPal will be discontinued but Centaury will be rebased up on Waterfox Classic, isn't it? Should you put more information into README about the future direction of this project? Thanks.

**Comment by Feodor2** [comment]
Yes but after mypal, which i am going to do upon newer firefox quantum for winxp.

**Comment by ghost** [comment]
> Yes but after mypal, which i am going to do upon newer firefox quantum for winxp.

Please clarify a bit more. Are you going to start with waterfox which forked from firefox 56? With the last Rust-free firefox 55? Or do you want to do with modern firefox which the latest version now is 93.0? Rust is a major problem on XP. I have never seen any port of firefox after Rust become a hard requirement available for XP, the most modern firefox for XP is the Rust-free firefox 55.

**Comment by Feodor2** [comment]
The effort is on mypal upon newer firefox 68-78 including rust. When i shall complete i would be to make centaury upon waterfax 56.

**Comment by the-r3dacted** [comment]
> The effort is on mypal upon newer firefox 68-78 including rust. When i shall complete i would be to make centaury upon waterfax 56.

I think using Firefox 60 might be better.
